### PR TITLE
[Doc] Fix document link

### PR DESCRIPTION
**Pull request by cir7**
## Motivation
[Doc] Fix document link

### CODE REVIEW

**Review by Dai-Wenxun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 [![Percentage of issues still open](https://isitmaintained.com/badge/open/open-mmlab/mmaction2.svg)](https://github.com/open-mmlab/mmaction2/issues)
 
 [📘Documentation](https://mmaction2.readthedocs.io/en/latest/) |
-[🛠️Installation](https://mmaction2.readthedocs.io/en/latest/get_started/installation.html) |
+[🛠️Installation](https://mmaction2.readthedocs.io/en/latest/get_started/installation/installation.html) |
```
Comment (hukkai) [inline README.md:29]: this link is not right?

File: README.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ It is a part of the [OpenMMLab](http://openmmlab.com/) project.
 
 MMAction2 depends on [PyTorch](https://pytorch.org/), [MMCV](https://github.com/open-mmlab/mmcv), [MMEngine](https://github.com/open-mmlab/mmengine), [MMDetection](https://github.com/open-mmlab/mmdetection) (optional) and [MMPose](https://github.com/open-mmlab/mmpose) (optional).
```
Comment (hukkai) [inline README.md:116]: link not working

File: README.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ It is a part of the [OpenMMLab](http://openmmlab.com/) project.
...
 
-Please refer to [install.md](https://mmaction2.readthedocs.io/en/latest/get_started/installation.html) for detailed instructions.
+Please refer to [install.md](https://mmaction2.readthedocs.io/en/latest/get_started/installation/installation.html) for detailed instructions.
```
Comment (hukkai) [inline README.md:117]: link not working

File: projects/README.md [hunk 4]
```diff
@@ -8,10 +8,10 @@ Here is an [example project](./example_project) about how to add your algorithms
 
 We also provide some documentation listed below:
 
-- [Contribution Guide](https://mmaction2.readthedocs.io/en/dev-1.x/notes/contribution_guide.html)
+- [Contribution Guide](https://mmaction2.readthedocs.io/en/latest/get_started/installation/contribution_guide.html)
```
Comment (hukkai) [inline projects/README.md:11]: link not working

**Review by hukkai — CHANGES_REQUESTED** [review]

**Review by hukkai — APPROVED** [review]

**Pull request merged**